Source/WebKit/NetworkProcess/storage/NetworkStorageManager.cpp

```diff
@@ -85,11 +85,11 @@ static HashMap<String, ThreadSafeWeakPtr<NetworkStorageManager>>& activePaths()
 
 static String encode(const String& string, FileSystem::Salt salt)
 {
-    auto crypto = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
-    auto utf8String = string.utf8();
-    crypto->addBytes(utf8String.span());
-    crypto->addBytes(salt);
-    return base64URLEncodeToString(crypto->computeHash());
+    Vector<uint8_t> utf8String { string.utf8().span() };
```

Comment:
This seems like it's becoming worse.  It adds an allocation and possible reallocation that wasn't there before.